auth/keycloak/config/realm-export.json

```diff
@@ -1933,12 +1933,19 @@
       "clientRole": false,
```

Comment:
Importing the new realm configuration requires refreshing the container and volume.  See the README.md for the steps.